app/assets/stylesheets/earthworks.css

```diff
@@ -134,26 +134,48 @@
   outline: none;
 }
 
+.search-q .input-group {
+  padding: .5rem
```

Comment:
This shouldn't be necessary since the `.search-query-form` (which contains this and only this element) has the same padding already.